prow/apis/prowjobs/v1/types.go

```diff
@@ -565,6 +565,8 @@ type DecorationConfig struct {
 	// This field will not override the existing ProwJob's PodSecurityContext.
 	// Equivalent to PodSecurityContext's FsGroup
 	FsGroup *int64 `json:"fs_group,omitempty"`
+	// BloblessFetch prevents fetching objects in implicit clonerefs.
```

Comment:
Shouldn't this just say "BloblessFetch tells prow to avoid fetching objects when cloning using the --filter=blob:none flag." like we do for the prowjob_customresourcedefinition.yaml?

Also, I think it would be better to put this just below (or above) the existing `SkipCloning` option on line 511.

Reply:
Done